packages/merkle-distributor/kpi-options-helpers/calculate-uma-tvl.ts

```diff
@@ -24,7 +24,7 @@ export async function calculateCurrentTvl() {
 export async function getAllFinancialContractsData() {
```

Comment:
modified this helper to include the collateral symbol. should not affect any other usage of this helper.